lib/CleantalkSP/SpbctWP/Scanner/ScannerQueue.php

```diff
@@ -1017,6 +1019,58 @@ public function file_system_analysis($offset = null, $amount = null, $path_to_sc
         return $output;
     }
 
+    /**
+     * Check crontab for scheduled scans
+     *
+     * @return array
+     * @psalm-suppress ForbiddenCode
+     */
+    public function os_cron_analysis() // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps
+    {
+        $scanning_stages_storage = new ScanningStagesStorage();
+        $scanning_stages_storage->converter->loadCollection();
+        $stage_data_obj = $scanning_stages_storage->getStage(OSCronAnalysis::class);
+
+        /*  */
+        $result = shell_exec('crontab -l');
```

Comment:
Add @ before calling `shell_exec` to avoid [possibly error notice](https://www.php.net/manual/en/function.shell-exec.php#refsect1-function.shell-exec-errors)